kubernetes-itests/pom.xml

```diff
@@ -31,14 +31,10 @@
 
   <dependencies>
     <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-engine</artifactId>
-      <scope>test</scope>
-    </dependency>
-    <dependency>
-      <groupId>org.junit.jupiter</groupId>
-      <artifactId>junit-jupiter-migrationsupport</artifactId>
-      <scope>test</scope>
+      <groupId>junit</groupId>
+      <artifactId>junit</artifactId>
+      <!-- Arquillian is still not compatible with Junit 5, so using Junit 4 here -->
+      <version>4.12</version>
```

Comment:
Could we use a property in the root pom.xml instead of a "hardcoded" value instead? e.g. `${junit.compatible-with-arquillian.version}`